File: read-links.py
with open('src/app/page.tsx', 'r', encoding='utf-8') as f:
    lines = f.readlines()

for i, line in enumerate(lines):
    if 'QUICK_LINKS =' in line:
        print("".join(lines[i-2:i+15]))
        break

File: read_links.js
const fs = require('fs');
const content = fs.readFileSync('src/app/jobs/[id]/page.tsx', 'utf8');
const lines = content.split('\n');
for(let i=0; i<lines.length; i++) {
    if(lines[i].includes('apply_url') || lines[i].includes('official_pdf_url') || lines[i].includes('official_source_url')) {
        console.log(`Line ${i}: ${lines[i]}`);
    }
}
